### ISSUE TITLE
RFC: "Each" iterator expressions with Env vars

### ISSUE THREAD

**Issue by cgmartin** [cgmartin/envhandlebars#1]
Request for comments on supporting Handlebars each block expressions via Environment variables. What conventions/options would users like to see? 

http://handlebarsjs.com/builtin_helpers.html#iteration
http://handlebarsjs.com/#paths

**Comment by cgmartin** [comment]
One thought...

Template:

```
{{#each PEOPLE}}{{NAME}}{{/each}}
```

ENV vars:

```
PEOPLE_0_NAME=foo
PEOPLE_1_NAME=bar
```

Internally we would look for the convention `{block}_\d+(_{childVar})?` and build corresponding object array.

**Comment by cgmartin** [comment]
The above is how it has been implemented, in `v1.1.0`.
